### DISCUSSION TITLE
No more manual steps !!!

### DISCUSSION THREAD

**Discussion by vRGB10** [MissingCore/Music#50] [Ideas]
I have recently come to know about this app through a yt video and I have installed it on my phone. The impression was really good until I have to put each file on a specific folder (music folder) to play mp3 files . 

Personally I have been using YMusic (not the youtube music) for 4-5 years and I haven't facing any issues on it. But as a fan of android customisation, I love to try new things. As I said, the first impression was very good to see it's minimal design and aesthetics. But you can't expect someone to do every step manually just to play some music, as you know there are many other apps available to fulfill his needs .

So, my point is fix this issue ASAP and a small request -- please add a music player widget (like in Nothing Kwgt pack) if possible.

**Comment by cyanChill** [comment]
`v1.0.0-rc.11` has been released and contains a new allowlist/blocklist feature that should let you select where music will be found (defaults to the `Music` directory on your device).

With widgets, its not a high priority as of now as it should work with Nothing's default Music widget.
